### ISSUE TITLE
Accessibility:List:  This LI does not have a ul element (without an ARIA-assigned role); ol element (without an ARIA-assigned role); 

### ISSUE THREAD

**Issue by puru-hk** [SAP/fundamental-ngx#3503]
Description: This LI does not have a ul element (without an ARIA-assigned role); ol element (without an ARIA-assigned role); an element with a role set to the value: list as a parent; or a ul element (without an ARIA-assigned role), ol element (without an ARIA-assigned role) or element with a role set to the value 'list’ with an aria-owns attribute set to the ID of the element in the same DOM

Code snippet for Case 1:
<li id="fdp-list-item-8" ng-reflect-tab-index="0" class="fd-list__item ng-star-inserted" tabindex="0" fd-list-item="" role="listitem" _ngcontent-qap-c383=""><span class="fd-list__title ng-star-inserted" title="Name1" aria-label="Name1" fd-list-title="" _ngcontent-qap-c383="">Name1</span></li>

Code snippet for Case 2:
<li id="fdp-list-item-0" ng-reflect-tab-index="0" class="fd-list__item ng-star-inserted" tabindex="0" fd-list-item="" role="listitem" _ngcontent-lyu-c399=""><span class="fd-list__title ng-star-inserted" title="Title1" aria-label="Title1" fd-list-title="" _ngcontent-lyu-c399="">Title1</span></li>

Expected: List items which are orphaned or not found within a container are often not rendered properly in assistive technology. This is especially problematic when more than one list grouping is on screen and adjacent to one another or when nested lists are not placed properly within their parent list item.
Ensure that whenever page elements are used proper list markup is provided.

screenshot:
case1 
![image](https://user-images.githubusercontent.com/32538291/94653884-e3739280-0319-11eb-8e93-e413f12944ad.png)

case 2
![image](https://user-images.githubusercontent.com/32538291/94653897-e9697380-0319-11eb-976d-9151d5457b6c.png)

**Comment by sKudum** [comment]
In platform li and ul are wrapped by components, by which we don't find the preceding ul /ol directly. for accessibility li element is given a role="listitem" and ul has to give role="list"

**Comment by sKudum** [comment]
Opt message is added to help on voice over.
